Bricks/file/file.h

```diff
@@ -54,12 +54,17 @@ namespace current {
 // Platform-indepenent, injection-friendly filesystem wrapper.
 struct FileSystem {
 #ifndef CURRENT_WINDOWS
-  static constexpr char PathSeparatingSlash = '/';
+  static constexpr char PathSeparator = '/';
```

Comment:
Why? It indeed is a slash, and it indeed is a single character. Why confuse the reader by removing the word "slash" from the name of the constant?